src/applications/dispute-debt/containers/ConfirmationPage.jsx

```diff
@@ -1,109 +1,64 @@
-import React, { useEffect, useRef } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import { format, isValid } from 'date-fns';
 import { useSelector } from 'react-redux';
-import { waitForRenderThenFocus } from 'platform/utilities/ui/focus';
-import { scrollTo } from 'platform/utilities/scroll';
-import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
 
-import GetFormHelp from '../components/GetFormHelp';
+import {
+  ConfirmationView,
+  ChapterSectionCollection,
+} from 'platform/forms-system/src/js/components/ConfirmationView';
+import NeedHelp from '../components/NeedHelp';
+
+export const ConfirmationPage = ({ route }) => {
+  const { formConfig } = route;
 
-export const ConfirmationPage = () => {
-  const alertRef = useRef(null);
   const form = useSelector(state => state.form || {});
-  const veteranEmail = useSelector(state => {
-    return state.user?.profile?.email || '';
-  });
-  const { submission, data = {} } = form;
-  const { fullName, selectedDebts = [] } = data;
-  const submitDate = submission?.timestamp;
+  const { submission } = form;
+  const { response, timestamp } = submission || {};
+
+  // no confirmation number is returned from the API currently
+  const { confirmationNumber = '' } = response || {};
 
-  useEffect(
-    () => {
-      if (alertRef?.current) {
-        scrollTo('topScrollElement');
-        waitForRenderThenFocus('h2', alertRef.current);
-      }
+  // Dropping reason chapter to better match designs and avoid extra repeating noise
+  const trimmedConfig = {
+    ...formConfig,
+    chapters: {
+      personalInformationChapter: {
+        ...formConfig.chapters.personalInformationChapter,
+      },
+      debtSelectionChapter: {
+        ...formConfig.chapters.debtSelectionChapter,
+      },
     },
-    [alertRef],
-  );
+  };
```

Comment:
Just wanted to note, I modified the `formConfig` to drop a chapter so we can match designs a little better and cut down on repeatedinformation